pkgs/development/python-modules/execnet/default.nix

```diff
@@ -44,8 +44,10 @@ buildPythonPackage rec {
     apipkg
   ];
 
+  # sometimes crashes with: OSError: [Errno 9] Bad file descriptor
+  doCheck = !isPyPy;
+
   nativeCheckInputs = [
-    py
```

Comment:
This is still required as `python310Packages.execnet` now fails to build with the following errors:

```                                                                                                   
FAILED testing/test_basics.py::TestSerializeAPI::test_bytesio[123] - AttributeError: module 'py' has no attribute 'io'                                                                                            
FAILED testing/test_basics.py::TestSerializeAPI::test_bytesio[42] - AttributeError: module 'py' has no attribute 'io'                                                                                             
FAILED testing/test_basics.py::TestSerializeAPI::test_bytesio[val2] - AttributeError: module 'py' has no attribute 'io'                                                                                           
FAILED testing/test_basics.py::TestSerializeAPI::test_bytesio[val3] - AttributeError: module 'py' has no attribute 'io'                                                                                           
FAILED testing/test_basics.py::test_subprocess_interaction[sys.executable] - AttributeError: module 'py' has no attribute 'code'                                                                                  
FAILED testing/test_basics.py::test_io_message[thread-sys.executable] - AttributeError: module 'py' has no attribute 'code'                                                                                       
FAILED testing/test_basics.py::test_popen_io[thread-sys.executable] - AttributeError: module 'py' has no attribute 'code'                                                                                         
FAILED testing/test_basics.py::test_stdouterrin_setnull[thread] - AttributeError: module 'py' has no attribute 'io'                                                                                               
FAILED testing/test_basics.py::test_exectask[thread] - AttributeError: module 'py' has no attribute 'io'                                                                                                          
FAILED testing/test_threadpool.py::test_pool_clean_shutdown[thread] - AttributeError: module 'py' has no attribute 'io'                                                                                           
FAILED testing/test_basics.py::test_rinfo_source[sys.executable] - AttributeError: module 'py' has no attribute 'code'                                                                                            
FAILED testing/test_basics.py::test_geterrortext[sys.executable] - AttributeError: module 'py' has no attribute 'code'                                                                                            
FAILED testing/test_basics.py::TestMessage::test_wire_protocol - AttributeError: module 'py' has no attribute 'io'                                                                                                
FAILED testing/test_basics.py::TestSourceOfFunction::test_function_without_known_source_fails - AttributeError: module 'py' has no attribute 'builtin'
FAILED testing/test_basics.py::TestGlobalFinder::test_local - AttributeError: module 'py' has no attribute 'code'
FAILED testing/test_basics.py::TestGlobalFinder::test_global - AttributeError: module 'py' has no attribute 'code'
FAILED testing/test_basics.py::TestGlobalFinder::test_builtin - AttributeError: module 'py' has no attribute 'code'
FAILED testing/test_basics.py::TestGlobalFinder::test_method_call - AttributeError: module 'py' has no attribute 'code'
```